src/app_engine/constants.py

```diff
@@ -15,7 +16,13 @@
 TURN_URL_TEMPLATE = '%s/turn?username=%s&key=%s'
 CEOD_KEY = '4080218913'
 
-WSS_HOST_PORT_PAIR = 'apprtc-ws.webrtc.org:443'
+WSS_HOST_PORT_PAIRS = ['apprtc-ws.webrtc.org:443', 'apprtc-ws-2.webrtc.org:443']
+WSS_HOST_STATUS_LIST_KEY = 'wss_host_status_list'
+
+WSS_HOST_INSTANCE_KEY = 'instance'
+WSS_HOST_STATUS_KEY = 'status'
```

Comment:
I find 'status' and 'status_code' are confusing. Maybe 'status' should be 'is_up' or 'is_healthy'